logging.md

```diff
@@ -0,0 +1,34 @@
+# Conventions for LOGGING
+[See issues](https://github.com/OCR-D/spec/issues/11)
+All logging may be stored in files. 
```

Comment:
It should be made clear that this is a different spec for different target audience than e.g. the [docs on the CLI](https://ocr-d.github.io/cli), which states

> Data printed to `STDERR` and `STDOUT` is captured linewise and stored as log data.

Maybe an introductory section, explaining the purpose of the document, who should read it and which are the essential requirements to implement.

Reply:
I'll write something.